.gitignore

```diff
@@ -7,3 +7,4 @@ old.txt
 *.test
 vendor/
 *.xml
+.idea
```

Comment:
This is unrelated to the PR (and probably belongs in a user level gitignore file, not every single project) please drop the commit that added it; thanks!